keras_contrib/layers/spatial_transformer_networks.py

```diff
@@ -0,0 +1,230 @@
+# -*- coding: utf-8 -*-
+from .. import backend as K
+from keras.engine import Layer
+
+if K.backend() == 'tensorflow':
```

Comment:
This code needs to go into the specific backend's python file. Also maybe try adding mesh grid to keras itself?

Reply:
you mean keras-team/keras?

Reply:
Fixed other issues, I will need a little more elaboration on this one.

Reply:
In the [newest implementation](https://github.com/oarriaga/STN.keras/blob/master/src/models/layers.py) there is now no need to have K_matmul defined for each backend.